qds_sdk/commands.py

```diff
@@ -136,6 +150,19 @@ def get_log(self):
         r = conn.get_raw(log_path)
         return r.text
 
+    def get_log_partial(self, err_pointer=0, tmp_pointer=0):
+        """
```

Comment:
@msumit: `get_log` is an internal function. We should be good as long as people are only calling this function. They won't have to change signature of this fn because the extra arguments (`err_pointer `, `tmp_pointer `) that we want to add, will have default values.